SDKTest/NeuroIDClassTests.swift

```diff
@@ -1392,7 +1392,7 @@ class NIDClientSiteIdTests: XCTestCase {
 
 class NIDSendTests: XCTestCase {
     func test_getCollectionEndpointURL() {
-        let expectedValue = "https://receiver.neuro-dev.com/c"
+        let expectedValue = "https://receiver.neuroid.cloud/c"
```

Comment:
Why did this change?

Reply:
this test broke. I don't know why but this is the fix for the test. Looks like it is using production to test?

Reply:
This was breaking locally for me but was passing in my PR.